lit_gpt/config.py

```diff
@@ -51,8 +53,9 @@ class Config:
     n_query_groups: Optional[int] = None
     shared_attention_norm: bool = False
     _norm_class: Literal["LayerNorm", "RMSNorm"] = "LayerNorm"
+    rmsnorm_add_unit_offset: bool = False
```

Comment:
Since you are using partial for rmsnorm, we can be sneaky and set this on the partial if `_mlp_class` is GemmaMLP, avoiding this argument

Reply:
Ok. 
Since for now only Gemma uses it (and maybe noone will ever reuse it in their models) we can do it.